operations/serviceordernotes.md

```diff
@@ -1,31 +1,33 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
 # Service order notes
 
-> **Service order:** This is the general name for an order made against a service, which includes both 'stay' service orders, called [Reservations](reservations.md#reservation), and 'product' service orders, which we simply call [Orders](orders.md). Operations such as [Get all service order notes](#get-all-service-order-notes) will accept Reservation IDs or Order IDs as service order identifiers.
-
 ## Get all service order notes
 
-Returns all notes associated with the given service orders. Service orders can be reservations or product orders. Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../concepts/multi-property.md).
+Returns all notes associated with the given service orders. Service orders can be reservations or product orders. Note this operation uses [Pagination](../guidelines/pagination.md) and supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+> **Service order:** This is the general name for an order made against a service, which includes both 'stay' service orders, called [Reservations](reservations.md#reservation), and 'product' service orders, which we simply call [Orders](orders.md). Operations such as [Get all service order notes](#get-all-service-order-notes) will accept Reservation IDs or Order IDs as service order identifiers.
```

Comment:
suggestion: Please check how this gets rendered in Gitbook, my concern is that you're missing a newline between the paragraph and a blockquote, which sometimes confuses parsers.

Reply:
I was checking this in visual code and that was fine. Not in the Gitbook but I've added a new line just in case, it should not have an effect.
![image](https://github.com/user-attachments/assets/f778862d-cb4a-4583-8341-603274c9db93)